starlite/datastructures/response_containers.py

```diff
@@ -59,7 +60,12 @@ class Config(BaseConfig):
 
     @abstractmethod
```

Comment:
why do you need to update the abstract method with the `request` kwarg? you can define it only for `Template` IMO.

Reply:
ok, i see why you need this value.